### PR TITLE
python: add geographiclib and update geopy

### PR DESCRIPTION
**Pull request by MMesch**
###### Motivation for this change

my first nixpkgs PR. Let me know if something should be different.

(1) added geographiclib from pip.
(2) updated geopy from 1.11 -> 1.18
(3) enabled python3 support for geopy

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/geopy/default.nix [hunk 1]
```diff
@@ -11,7 +11,7 @@
 buildPythonPackage rec {
   pname = "geopy";
   version = "1.18.0";
-  disabled = !isPy27;
+  disabled = false;
```
Comment (prusnak) [inline pkgs/development/python-modules/geopy/default.nix]: Maybe just remove the line completely?

File: pkgs/development/python-modules/geographiclib/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, isPy27
+}:
+
+buildPythonPackage rec {
+  pname = "geographiclib";
+  version = "1.49";
+  disabled = false;
```
Comment (dotlambda) [inline pkgs/development/python-modules/geographiclib/default.nix]:
```suggestion
```

File: pkgs/development/python-modules/geographiclib/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "635da648fce80a57b81b28875d103dacf7deb12a3f5f7387ba7d39c51e096533";
+  };
+
+  doCheck = false;  # too much
```
Comment (dotlambda) [inline pkgs/development/python-modules/geographiclib/default.nix]: Too much what? Please add a meaningful comment

File: pkgs/development/python-modules/geographiclib/default.nix [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+  meta = with stdenv.lib; {
+    homepage = "https://geographiclib.sourceforge.io";
```
Comment (dotlambda) [inline pkgs/development/python-modules/geographiclib/default.nix]:
```suggestion
    homepage = https://geographiclib.sourceforge.io;
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Review by dotlambda — CHANGES_REQUESTED** [review]
The last two commits should be squashed together using the commit message `pythonPackages.geopy: 1.11 -> 1.18`

File: pkgs/development/python-modules/geographiclib/default.nix (hunk 3 above)
Comment (prusnak) [reply pkgs/development/python-modules/geographiclib/default.nix]: This was copy-pasted from geopy package. It probably shouldn't belong in this newly created file.

**Comment by knedlsepp** [comment]
@MMesch Are you still intersted in pushing this forward? Looks like we could use this for #56826.

**Comment by FRidh** [comment]
Nothing happening so closing.

**Pull request closed without merging**